src/register.js

```diff
@@ -0,0 +1,27 @@
+export function findProduct(bikes, code) {
+    for(let i = 0; i < bikes.length; i++) {
+        const bike = bikes[i];
+        if(bike.code === code) {
+            return bike;
+        }
+    }
+    return null;
+}
+
+export function calcLineTotal(quantity, price) {
+    return Number((quantity * price).toFixed(2));
+}
+
+export function calcOrderTotal(cart, bikes) {
+    // declare order total variable
+    let orderTotal = 0;
+    // loop the array of cart (line items)
+    for(let i = 0; i < cart.length; i++) {
+        const cartQty = cart[i];
```

Comment:
best practice: 
const cartQty = cart[i].quantity;

instead of dot notating when you pass params into the function.
saw you did this a few times in this file and in your tests.